CHANGES.rst

```diff
@@ -1,6 +1,24 @@
 Changes
 =======
 
+``v0.4.0``
```

Comment:
and let's directly update to `v0.4.1` as `v0.4.0` was already released on Zenodo
```suggestion
``v0.4.1``
```